templates/registration/activation_email.html

```diff
@@ -1,24 +1,23 @@
-<div style="display:block;margin: 0 2px;padding: 1em;border: 3px solid #FFDE05;background-color: #000;border-radius: 6px;font-size: .95em;color: #444;margin-bottom:0.75em">
-<div style="display:table;margin-left:2em"><img src="https://avatars1.githubusercontent.com/u/6934864?v=3&s=101" style="display:inline;vertical-align: middle">
-<h1 style="font-size:4em;display:inline;vertical-align: middle"><a href="//{{ site.domain }}" style="text-decoration:none;color:gray"><span style="color: #FFDE05">DM::</span>OJ</a>
-      </h1>
-</div>
-</div>
-<div style="display:block;margin: 0 2px;padding: 1em;border: 3px solid #2980B9;background-color: #f8f8f8;border-radius: 6px;font-size: .95em;color: #444;">
+<b>Thanks for registering on the {{ site.name }}! We're glad to have you.</b>
+<br><br>
+The last step is activating your account. Please activate your {{ SITE_NAME }} account in the next {{ expiration_days }} days.a
+<br><br>
+Please click on the following link to activate your account:
+<p style="margin-left:1em">
+      <a href="http://{{ site.domain }}/accounts/activate/{{ activation_key }}/">http://{{ site.domain }}/accounts/activate/{{ activation_key }}</a>
```

Comment:
We cannot use `http`. Pass the scheme as a variable. We should have it somewhere.